### PR TITLE
Box new architecture

### PR DESCRIPTION
**Pull request by gshriki**
Box with new architecture.

Line with `brandData = {...` needs to be removed before merge.

### CODE REVIEW

**Comment by meirwah** [comment]
@Spivaka are you reviewing this?

File: Integrations/integration-Box.yml [hunk 1]
```diff
@@ -6,193 +6,327 @@ display: Box
...
...BxPnWNEf6iQxa5biPLAZhX56aGyfau5sql4hePbjhUxGwB83nVLAXoH1p4bb5i6zXUIUSn5cDuXVg4Bh6RPVRYmItW/y8XFOACkYYu09FTDZL4L4FoAk1z3EI0mAXoVuMvLBH7Ot8lEboTj6fli9AYoFgEwrnuIRpMAL1rVm3vLQw89dZ5kXffQyAvHUyeJ4GkAx7puKSA7PZH5zfXVr7sOISpFi9bs+ouxfX2/BfAl1y0FREX16pZIzapcXZADQMqZaGxTWbZs0qWiegOAWa57iEbYb1XlF4Hs7rbmxpn9rmOICFjS9l7IU2+ZqHwHPDmYilsWwIOwuC3eEHrJdQyNvmh71wzft48D+KzrlgKw2xezoK2+aqPrEKJS9uEQ8AkAp7tuKQB9AlzaEg615vKiHABSzjU1qXl1Troegu8KMB/8PaPi0QcgBmvu4Im+RPmrtiM1LpjVywD5FoAvuu4hyqHdInjAWtyViIQ2u44htzgEHJQdYrC4pS70gusQIgJqO1KTg1lpB/Qs1y15S3BIrV6aiNS05f7SRCOo4aGtJ6jnXQFgOYBqxzlEQ/WOit4V6Dermhure1zHENHgRVq6zlRjvy1AFMAY1z1EQ6CArhPgngnv9yfuu2LGIddBlD+ise4qG8w+psBs1y35RoE3Ap5Z1LykKum6hYj+JBrbOtYPeL+CIOy6JQ/thZG6eF31upG4OAeANCoWrNPAsXtSi9SaqyF6MYCA6yaiwxIcUkXCiK6a/UroyaYmsa6TiGjoorGtk/yguRyQKwDMdd1D9OkkBdX7rHirWsOVW1zXUP6KxnYfY4OHVikQcd2SRzao2iWJyNRdrkOI6OOamtRsPDV9qyquc92SLxR4A2qWJiJVr43UPTgApFEXaempVq//r0XlSvAEM8ozAryoIqv6Av0PPrz4uF7XPUSUew1t2+eqNVcAejl4eBXllywgj4jY/9h1TOgRHupBg6YqDYn091TwUwBB1zlOKe7xsv51zY3TDrpOIaLDC8dTfyOCWwCMc93ikgiaMxK8qr2uYu+I3mckL050WKoSaU+fCx9XQRABcLTrJCpZ3Qo8aIy/qqVuWqfrGCIaHYvWvDVmbP+4ekCWQ/FVAJ7rJipRKp0Q/XXWZP+zve4zKdc5VLjC8dQ5AvwaghrXLQ7shOCaeH0o4TqEiAav4aHUF+DhAQVOc93iQ
```
Comment (Spivaka) [inline Integrations/integration-Box.yml]: remove this before merging. your words @gshriki :)

Comment (Spivaka) [inline Integrations/integration-Box.yml]: would not do this here as this is command specific, right now you are adding it to all commands

File: Integrations/integration-Box.yml [hunk 2]
```diff
@@ -321,4 +462,4 @@ script:
       folders. To move everything from the root folder, use “0” which always represents
       the root folder of a Box account (Currently only moving of the root folder (0)
       is supported)
-system: true
```
Comment (Spivaka) [inline Integrations/integration-Box.yml]: Think this should not change

File: Integrations/integration-Box.yml (hunk 1 above)
Comment (Spivaka) [inline Integrations/integration-Box.yml]: I think a fromversion is needed, make sure with @aviadl

**Pull request merged**